tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/Announcements.java

```diff
@@ -223,7 +223,7 @@ public Response submitAnnouncement(
         // Only TC or Admin can specify orgs other than ORGS_MINE
         if (!UserRegistry.userIsManagerOrStronger(session.user)
                 || (!ORGS_MINE.equals(request.orgs)
-                        && !UserRegistry.userIsTC(session.user))) { // userIsTC means TC or stronger
+                        && !UserRegistry.userIsTCOrStronger(session.user))) {
```

Comment:
One can only add a comment like this so many times before the need to rename the function becomes irresistible